### ISSUE TITLE
Does enabling a syntax for Alias make sense?

### ISSUE THREAD

**Issue by llinfeng** [vimwiki/vimwiki#334]
## Feature request: introduce a syntax for creating `alias`?

Over the time, I start to loose concise memory of what my wikis are named. When I created them, indeed I was using what first came to mine **at the time**. Yet, as time goes by, that name no longer strike me first. For example, I have kept my user notes for `Total-Command` (a superb file manager on Windows platform) in a Wiki, and named it `TC.wiki` years ago. I think I must have been more proud of the file manager years ago. 

I am aware that there is a "rename" function that can cast the Wiki (`:VimwikiRenameLink`, together with all hyperlink in the local Wiki directory to a new name. However, this renaming process takes quite a while.

A simple `alias` method/syntax/entry-in-config-file may help improve.

## my current solution
I am now creating alias wiki with a single shortcut pointing to the __correct__ Wiki, and it takes an additional keystroke or two to get to the correct place.

**Comment by Nudin** [comment]
When you write name of a wiki, I suppose you mean name of a wikipage, right?
Couldn't you just use a symlink?

**Comment by llinfeng** [comment]
I read the notice and found this old thread of mine. Over the years, I have been using an "index page" to help me keep track of the wiki-pages (`*.wiki` or `*.md` files, indeed).

`symlink` can be a challenge for my setup, where I use Windows + Linux systems across several physical machines. I keep my personal wiki folder in-sync through a cheaper alternative of Dropbox, called [Nutstore](https://www.jianguoyun.com/). For this Dropbox-like situation, it is unclear how to operationalize the good number of machines in-sync for its `symlink` settings:
* Should I issue `symlink` command on one machine? If so, should this be the 24/7 Windows machine or the Linux machine ...
* Should I issue `symlink` command on all machines?
* Is it possible, if at all, to "sync" the `symlink` mappings/settings across machines :)

Based on previous attempts a while ago, I should have given it a try and the `sync-client` should have successfully messed up the pair of files that were `symlink`-ed ==> while setting it up on one machine through the command-line is totally legitimate, the `sync-client` does not think that way: it treated both "files" as _real files_ and should try to sync both of them. I do observe a good number of conflict files, as I may recall now.

----

For now, I keep labeling the "index page" and have started to explore the `tag`s. Alias of files/wiki-pages is no longer necessary.

**Comment by llinfeng** [comment]
Dropbox now officially supports symlink files. Yet, `ln -s` only creates a symbolic file that is reusable on other Linux machines. Currently, 2021-07-15, the `mklink` command on Windows shall create an alias file that is reusable on both Windows and Linux machines. Here is an example:

```
ln -s kvm.md barrier.md   #this is for Linux
mklink barrier.md kvm.md #this is for Windows PowerShell (or Command Prompt)
```

Supposedly, either line above should create an "alias file", named "barrier.md", that points to "kvm.md". Yet, only the second line actually gets synced across all my Linux and Windows machines. The first line, `ln -s`, only creates a symbolic file with properties `lrwxrwxrwx` that is exclusively available to other Linux machines.

-----

In short, to create an "effective alias" that can be reused for both Linux and Windows machines through Dropbox, the `mklink` command on Windows is the way to go. (Not tested on macOS.)
